index.js

```diff
@@ -1,205 +1,13 @@
-              "\x1b[31m%s\x1b[0m",
-              `Options(Config) is not a not valid JSON Fallback activate`
-            );
-            program.config = false;
-            console.log(e);
-          }
-        }
-        if (program.config && validateConfig(config, options)) {
-          console.log("Now using Options object");
-          Object.keys(options).forEach(function(input) {
-            options[input] = config[input];
-          });
-        } else {
-          // If inputs exist, iterate through them and prompt for values.
-          Object.keys(options).forEach(function(input) {
-            var value = readlineSync.question(
-              "[" +
-              step +
-              "]: Enter a value for " +
-              input +
-              " (" +
-              options[input].type +
-              ", default: " +
-              options[input].default +
-              "): "
-            );
-            options[input] = value;
-          });
-        }
-        // Add the step and its inputs to the sequencer.
-        sequencer.addSteps(step, options);
-      });
-
-      var spinnerObj = { succeed: () => { }, stop: () => { } };
-      if (!process.env.TEST)
-        spinnerObj = Spinner("Your Image is being processed..").start();
-
-      // Run the sequencer.
-      sequencer.run({ progressObj: spinnerObj }, function() {
-        // Export all images or final image as binary files.
-        sequencer.exportBin(program.output, program.basic, outputFilename);
-
-        //check if spinner was not overriden stop it
-        if (!spinnerObj.overrideFlag) {
-          spinnerObj.succeed();
-          console.log(`\nDone!!`);
-        }
-      });
-    });
-  });
-
-  // Takes an array of steps and checks if they are valid steps for the sequencer.
-  function validateSteps(steps) {
-    // Assume all are valid in the beginning.
-    var valid = true;
-    steps.forEach(function(step) {
-      // If any step in the array is not valid (not a property of modulesInfo), set valid to false.
-      if (!sequencer.modulesInfo().hasOwnProperty(step)) {
-        valid = false;
-      }
-    });
-
-    // Return valid. (If all of the steps are valid properties, valid will have remained true).
-    return valid;
-  }
-
-  //Takes config and options object and checks if all the keys exist in config
-  function validateConfig(config_, options_) {
-    options_ = Object.keys(options_);
-    if (
-      (function() {
-        for (var input in options_) {
-          if (!config_[options_[input]]) {
-            console.error(
-              "\x1b[31m%s\x1b[0m",
-              `Options Object does not have the required details "${
-              options_[input]
-              }" not specified. Fallback case activated`
-            );
-            return false;
-          }
-        }
-      })() == false
-    )
-      return false;
-    else return true;
-  }
-}
-
+//JavaScript I have Coded Starts from Here.........................

```

Comment:
It looks like you could open a new PR with only this code in `/examples/lib/scrollToTop.js` -- we can help you once you've isolated the code like this! Don't worry, you'll get it! 👍 

Thanks for your help!

Reply:
You could try to work in this PR but I think it might be a cleaner approach to start with a fresh copy where you haven't modified all the other files yet. But be careful not to lose your work as you do this!

Reply:
I still don't understand should I make changes in the /examples/lib/scrollToTop.js file or a separate file like I have done before?

Reply:
If you are asking for a fresh copy I submitted one before, a new js file.